### ISSUE TITLE
Enums shown as global

### ISSUE THREAD

**Issue by srad** [jsdoc3/jsdoc#970]
Im using JSDoc 3.3.0-beta3 and the following enums are shown as "global" members. Example:

``` javascript
define(function () {
    /**
     * @exports some/type
     * @enum {string}
      */
    var exports = {
        A: 'abc'
    };

    return exports;
});
```

**Comment by hegemonic** [comment]
Thanks for filing this as a new issue.

It looks like the presence of the `@enum` tag is preventing the `@exports` tag from doing its thing. I'll investigate.

**Comment by hegemonic** [comment]
This appears to be a viable workaround:

```js
/** @module some/type */

define(function () {
    /**
     * @alias module:some/type
     * @enum {string}
      */
    var exports = {
        A: 'abc'
    };

    return exports;
});
```

**Comment by srad** [comment]
```
define(function () {
    /**
     * @alias module:some/type
     * @namespace
     * @property {string} A
      */
    var exports = {
        A: 'abc'
    };

    return exports;
});
```
But I'm not sure right now, if I can reference the members via ```@link```.

**Comment by hegemonic** [comment]
Fixed on master. The fix will be included in JSDoc 3.6.0.
